testpilot/frontend/templates/testpilot/frontend/index.html

```diff
@@ -24,7 +24,9 @@
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-    ga('create', 'UA-49796218-34', 'auto');
+    if (typeof(ga) !== 'undefined') {
+        ga('create', 'UA-49796218-34', 'auto');
+    }
```

Comment:
Would maybe be nice to those an `else console.warn('You have google analytics blocked. We understand. Take a look at our privacy policy to see how we handle your data.')`